pydundas/rest/constant.py

```diff
@@ -0,0 +1,169 @@
+class ConstantIdError(KeyError):
+    """Constant key does not exist."""
+    pass
+
+
+class ConstantNameError(KeyError):
+    """Constant name does not exist."""
+    pass
+
+
+class Constant:
+    """Expose Dundas constants with human names"""
+
+    # In a javascript console:
+    # for (d in dundas.constants) { if (d.endsWith('_ID')){console.log(d + ": " + dundas.constants[d])}}
```

Comment:
I notice some constants that don't match the predicate: `endsWith('_ID')`. Should these be included in this comment somehow? Or should they be removed from the list below?